io/xml/src/TXMLEngine.cxx

```diff
@@ -286,19 +286,20 @@ class TXMLInputStream {
    ////////////////////////////////////////////////////////////////////////////
    /// Allocate more data for the buffer, preserving content
 
-   Bool_t ExpandStream()
+   Bool_t ExpandStream(char **curr = nullptr)
```

Comment:
All but one call to `ExpandStream` pass `curr`. Can you just force `ExpandStream(char* &curr)`?

Reply:
But at that place I do not have any extra pointer which should be recalculated.
Shoud I create dummy for that?

Reply:
In my personal preference, yes - I hate `**`! :-) But your decision, I was merely suggesting this (in that single case, you could move the declaration of `curr` up, before the call to `ExpandStream(curr)`). Let me know what you prefer and I'll merge.